### DISCUSSION TITLE
Epic Games: 3rd Party Auth Request

### DISCUSSION THREAD

**Discussion by Julianwan** [supabase/supabase#2262] [Feature Requests]
I searched and didn't see any topic about Auth provider requests...

Are there any plans to implement support for additional providers?

https://dev.epicgames.com/docs/services/en-US/Interfaces/Auth/index.html

I would really love to see EPIC Games included

**Comment by w3b6x9** [comment]
@Julianwan we're continuing to add additional auth providers so you can create an issue in https://github.com/supabase/gotrue and then we or the community will implement over time.
